src/register/TermsAndPrivacyPolicyModal.jsx

```diff
@@ -0,0 +1,903 @@
+import React, { useState } from "react";
+import { Modal, Button, Tabs, Tab } from "react-bootstrap";
+
+const TermsAndPrivacyPolicyModal = ({ show, handleClose }) => {
+  const [activeKey, setActiveKey] = useState("code-of-conduct");
+  return (
+    <Modal className="terms-and-privacy-modal" show={show} onHide={handleClose}>
+      <Modal.Header closeButton></Modal.Header>
+      <Modal.Body>
+        <Tabs activeKey={activeKey} onSelect={(k) => setActiveKey(k)}>
+          <Tab eventKey="code-of-conduct" title="Code of Conduct">
```

Comment:
@Rahulm2310 , please replace the content for the Code of Conduct to [this one](https://github.com/anitab-org/documentation/blob/master/Contributing/CODE_OF_CONDUCT.md) from the Code of Conduct under anitab-org/documentation. The one you provide here is specific to GHC code of conduct only.